server/routes/projects.js

```diff
@@ -0,0 +1,55 @@
+const { ObjectId } = require('bson');
+const express = require('express');
+const projectsData = require('../data/projects');
+const {
+	sendErrResp,
+	isValidArray,
+	isValidObj,
+	isValidStr,
+} = require('../utils');
+const { isValidProjectName } = require('../utils/projects');
+const router = express.Router();
+const { authenticateToken } = require('../middleware/auth');
+
+router.route('/project').post(authenticateToken, async (req, res) => {
+	const { user } = req;
+	const {
+		name,
+		description,
+		github,
+		media,
+		technologies,
+		owner,
```

Comment:
the project request body will not have an owner object. we create that based on the current user

Reply:
Ok got it.. will update this using the middleware function